packages/front/svelte/views/association/components/Bodacc/Announcement.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+    import Callout from "@dsfr/Callout.svelte";
+    import AnnouncementController from "./Announcement.controller";
+
+    export let announcement;
+
+    const controller = new AnnouncementController(announcement);
+</script>
+
+<div class="announcement">
+    <Callout title={announcement.familleavis_lib} titleSize="5" href={controller.url} labelAction="Voir l'annonce">
+        <div>
+            <p class="fr-text--lead fr-mb-2v">{announcement.dateparution} - {announcement.typeavis_lib}</p>
+            <p class="fr-text--lead fr-mb-2v">
+                <b>Annonce déposée au :</b>
```

Comment:
J'ai en tête que c'est une mauvaise pratique d'utiliser `<b>`
```suggestion
                <span class="fr-text--bold">Annonce déposée au :</b>
```

Reply:
Je trouve ça très verbeux. Tu as une référence qui en parle ? C'est quand même utile d'utiliser les balises HTML (mais à vérifier si le CSS du DSFR surcharge bien <b> sur du `fr-text--bold`

Reply:
`font-weight: bold;` (associé par défaut à `<b>` est à `700`comme le `fr-text--bold`. J'ai envoyé un message sur le slack DSFR pour voir ce qu'ils recommande.

Au pire on peut créer un composant <Bold /> qui applique la classe pour plus de lisibilité dans le HTML 

Mais j'ai aussi lu, et je suis d'accord, `Always use HTML tags when they can add meaning, structure, or semantics to your content.`. Ceci dit, je peux remplacer `<b>` par `<strong>` ou `<em>` pour une meilleure accessibilité ? Je ne sais pas à quel point ça change la lecture de la page

Reply:
Pour l'instant et afin de merge la PR j'ai ajouté dans `global.css` un `font-weight: 700 !important;` pour s'assurer que <b> équivaut au `fr-text--bold` qui correspond à la taille standard du "gras"